ui-tests/cypress/integration/administration/languages.spec.js

```diff
@@ -179,6 +179,20 @@ describe('Languages', () => {
         getLanguageTableRowByCode(languages.en.code).should('exist').and('contain', '*');
     });
 
+    it([Tag.FEATURE, 'ENG-3237'], 'Verify the available languages in the language selector', () => {
+        let expectedLanguagesInDropdown = Object.keys(languages).length-1;
+        currentPage = openLanguagesPage();
+        currentPage.getContent().getLanguageDropdown().as('languageDropdown');
+        currentPage.getContent().getLanguageDropdown().children(htmlElements.option).should('have.length', expectedLanguagesInDropdown);
+        cy.get('@languageDropdown').then(res => {
```

Comment:
I resorted to using a label here, because this way the cycle takes about 30 seconds, compared to having cypress make a new query each time (it took almost 4 minutes in total)